app/controllers/spid_rails/spid_controller.rb

```diff
@@ -52,9 +53,8 @@ def sp_attributes
 
   def idp_attributes
     parser = OneLogin::RubySaml::IdpMetadataParser.new
-    parser.parse_remote_to_hash idp_xml(params[:idp]),
```

Comment:
@Jecko-o Questo File.open(Rails.root+'public/metadata-idp-gov.xml') è momentaneo?

Reply:
si quasi tutto quello che c'èdentro spid controller è prossimo a essere rimosso in favore di metodi nelle classi metadata/request/response